crates/agent/src/integration_tests/harness.rs

```diff
@@ -168,7 +168,28 @@ impl TestHarness {
                     'materialization',
                     'http://test.test/',
                     '{"type": "object"}',
-                    '{"type": "object", "properties": {"id": {"type": "string", "x-collection-name": true}}}',
+                    '{"type": "object", "properties": {"id": {"type": "string", "x-collection-name": true}, "schema": {"type": "string", "x-schema-name": true}, "delta": {"type": "boolean", "x-delta-updates": true}}}',
+                    '{/id}',
+                    '{"type": "success"}'
+                ) on conflict do nothing
+            ),
+            materialize_tag_no_annotations as (
```

Comment:
nit: this could be added as just another value in the `materialize_tag` cte

Reply:
the rows that are inserted are accessible by these assigned names in the rest of the script, and I wanted to keep both of these rows accessible with a name